bundles/net.tourbook/src/net/tourbook/database/TourDatabase.java

```diff
@@ -5842,24 +6000,26 @@ private boolean updateDb__2_Data(final Connection conn, final SplashManager spla
 
       try {
 
-         updateDb_003_To_004_DataUpdate(conn, splashManager);
-         updateDb_004_To_005_DataUpdate(conn, splashManager);
-         updateDb_008_To_009_DataUpdate(conn, splashManager);
-         updateDb_010_To_011_DataUpdate(conn, splashManager);
-         updateDb_012_To_013_DataUpdate(conn);
-         updateDb_019_To_020_DataUpdate(conn, splashManager);
-         updateDb_021_To_022_DataUpdate(conn, splashManager);
-         updateDb_022_To_023_DataUpdate(conn, splashManager);
-         updateDb_024_To_025_DataUpdate(conn, splashManager);
-         updateDb_027_To_028_DataUpdate(conn, splashManager);
-         updateDb_028_To_029_DataUpdate(conn, splashManager);
-         updateDb_031_To_032_DataUpdate(conn, splashManager);
-         updateDb_033_To_034_DataUpdate(conn, splashManager);
-         updateDb_036_To_037_DataUpdate(conn, splashManager);
-         updateDb_039_To_040_DataUpdate(conn, splashManager);
-         updateDb_041_To_042_DataUpdate(conn);
-         updateDb_042_to_043_DataUpdate(conn, splashManager);
-         updateDb_046_to_047_DataUpdate(conn, splashManager);
+         updateDb_003_To_004_DataUpdate(connection, splashManager);
```

Comment:
@wolfgang-ch 
Note that I haven't refactored all the previous data updates because first, before I spent all this time refactoring those, I want to make sure that the refactoring I did is what you asked.

Reply:
I would not refactore an existing very critical code, then you also have to test it very thoroughly **->** so I would also not refactor `TourDataUpdate_046_to_047`

The older I am, the more I do "Don't touch a running system"